R/integrate_profile.R

```diff
@@ -136,6 +115,28 @@
 #' Columns mt and rt in the output dataframe provides migration traffic as a numeric value equal to
 #' migration traffic and reflectivity traffic from the start of the time series up till the moment of the time stamp
 #' of the respective row.
+#'
+#' #' \subsection{Ground speed (ff) and ground speed components (u,v)}{
+#' The height-averaged ground speed is defined as:
+#'
+#' \deqn{ff = \sum_i dens_i ff_i / \sum_i dens_i}{ff = \sum_i dens_i ff_i / \sum_i dens_i}
+#' with the sum running over all altitude layers between \code{alt_min} and
+#' \code{alt_max}, \eqn{dens_i} the bird density, \eqn{ff_i} the ground speed at
+#' altitude layer i.
+#'
+#' the height-averaged u component (west to east) is defined as:
+#'
+#' \deqn{u = \sum_i dens_i u_i / \sum_i dens_i}{u = \sum_i dens_i u_i / \sum_i dens_i}
+#'
+#' the height-averaged v component (south to north) is defined as:
+#'
+#' \deqn{v = \sum_i dens_i v_i / \sum_i dens_i}{v = \sum_i dens_i v_i / \sum_i dens_i}
+#' }
+#'
+#' Note that \eqn{ff_i=\sqrt(u_i^2 + v_i^2)}, but the same does not hold for the
+#' height-integrated speeds, i.e. \eqn{ff != \sqrt(u^2 + v^2)} as soon as the
+#' ground speed directions vary with altitude.
+#'
 #' }
```

Comment:
**Documentation**
- Add more comments in the code
- Reorder the subsection in the same order than the Available quantities subsection
- Correct the `\alpha `and `\pi`